src/locales/fr/dict.json

```diff
@@ -81,5 +81,87 @@
         "title": "Médias"
       }
     }
+  },
+  "header": {
+    "assure": "Assurer",
+    "carSubscription": "Auto-Abo",
+    "estimate": "Estimer",
+    "login": "Rapport",
+    "magazine": "Magazin",
+    "more": "Di più",
+    "search": "Rechercher",
+    "searchMenu": {
+      "additional": "Autres",
+      "advancedSearch": "Rechercher point de vente",
+      "dealer": "points de vente",
+      "findPartsAndAccessories": "Pièces et accessoires",
+      "insuranceComparison": "Comparaison d’assurances",
+      "recentSearches": "Dernières recherches",
+      "searchCars": "Rechercher des véhicules",
+      "searchMerchant": "Rechercher point de vente",
+      "searchMotorcycles": "Rechercher des motos",
+      "simpleSearch": "Recherche simple (page d'accueil)",
+      "vehicleRating": "Evaluer le véhicule",
+      "vehicles": "Véhicules",
+      "viewedListings": "Annonces vues"
+    },
+    "sell": "Vendre",
+    "userMenu": {
+      "accountSettings": "Paramétrage du compte",
+      "advertisePartsAccessories": "Insérer des pièces détachées",
+      "amagServiceLifePool": "AMAG Pool",
+      "autoRadar": "AutoRadar",
+      "b2bPlattform": "Plate-forme B2B",
+      "bookmarks": "Favoris",
+      "boosteras24": "Booster as24",
+      "businessImage": "BusinessImage",
+      "carPark": "Parc de véhicules",
+      "changePassword": "Changer le mot de passe",
+      "cockpit": "Cockpit",
+      "contactDetails": "Informations de contact",
+      "contactRequests": "Demandes de contact",
+      "createAd": "Créer une annonce",
+      "dealerDashboard": "DealerDashboard",
+      "dealerInfoSystem": "DealerInfoSystem",
```

Comment:
Nope, those are the names for the links we already use in production. I can check if we need to translate that, but I assume not because we use them like that on production.